pkg/ocm/clusters.go

```diff
@@ -410,6 +412,42 @@ func (c *Client) UpdateCluster(clusterKey string, creator *aws.Creator, config S
 		)
 	}
 
+	if config.HTTPProxy != nil || config.HTTPSProxy != nil || config.AdditionalTrustBundleFile != nil {
+		clusterProxyBuilder := cmv1.NewProxy()
+		if config.HTTPProxy != nil {
+			if *config.HTTPProxy == "" {
+				// user pressed the 'enter' button. No change is required.
+				config.HTTPProxy = nil
+			} else if *config.HTTPProxy == DoubleQuotesToRemove {
```

Comment:
You should not have to compare to `DoubleQuotesToRemove` here - that's why you have pointers already: if the user entered double quotes, then you set the value to an empty string, if the user didn't do anything, then you send `nil`. The point is that the double-quotes is an artifact of the interactive mode and the `ocm` package should not be aware of it.

Reply:
I must do that, otherwise, if I remove the check to double quotes, cs will get """" (double double quotes)

Reply:
No, I mean that in the `cmd/edit/cluster/cmd.go` file, you should be checking and replacing this:
```go
if httpProxy != nil {
  if *httpProxy == ocm.DoubleQuotesToRemove {
    *httpProxy = ""
  }
  clusterConfig.HTTPProxy = httpProxy
}
```
This means that double quotes do not need to be checked outside of the `cmd` file (maybe in the interactive validation function, but certainly not in the `ocm` package)

Reply:
@vkareh ok, I think I can do something. Still trying....